functions/webhooks/chatbotCallback.protected.ts

```diff
@@ -124,22 +117,35 @@ export const handler = async (
             attributes: JSON.stringify(releasedChannelAttributes),
           }),
           // Move control task to complete state
-          client.taskrouter.v1
-            .workspaces(context.TWILIO_WORKSPACE_SID)
-            .tasks(channelAttributes.controlTaskSid)
-            .update({ assignmentStatus: 'completed' }),
+          (async () => {
+            try {
+              await client.taskrouter.v1
+                .workspaces(context.TWILIO_WORKSPACE_SID)
+                .tasks(channelAttributes.controlTaskSid)
+                .update({ assignmentStatus: 'completed' });
+            } catch (err) {
+              console.log(err);
+            }
+          })(),
           // Remove this webhook from the channel
           channel
             .webhooks()
             .get(channelAttributes.channelCapturedByBot.chatbotCallbackWebhookSid)
             .remove(),
           // Trigger the next step once the channel is released
-          nextAction,
+          nextAction(),
         ]);
 
         console.log('Channel unblocked and bot session deleted');
       }
 
+      // Why we need to send this message?
```

Comment:
The always want to send the Lex response to the channel, so the user can see it. The reason it is moved after all the other code is that `nextAction` now adds a webhook with a Studio trigger for pre survey, and the message being sent after will trigger the "second execution" of the flow so the channel triggers a task for Flex and all of that.

Reply:
Sorry, I've forgotten to remove this comment😅

I've noticed it later on. I just got confused initially when I was getting code from the Lex pivot stuff.
I'll remove the comment